src/test/java/se/knowit/bookitregistration/controller/RegistrationControllerTest.java

```diff
@@ -128,7 +128,7 @@ void deleteByIdRequest_Should_DeleteRegistration_And_Return_204_() throws Except
         mockMvc.perform(
                 delete(PATH + "/" + DEFAULT_REGISTRATION_UUID))
                 .andExpect(status().isNoContent());
-        verify(registrationService, times(1)).delete(DEFAULT_REGISTRATION_UUID.toString());
+        verify(registrationService, times(1)).deleteByRegistrationId(DEFAULT_REGISTRATION_UUID.toString());
```

Comment:
You can skip "times(1)" here, that is the default for the verify method

Reply:
Ok!